app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,28 @@
+class VotesController < ApplicationController
+  def create
+    @review = Review.find(params[:review_id])
+    @vote = Vote.new
+    @vote.review = @review
+    @vote.user = current_user
+    @superpower = @review.superpower
+    @reviews = @superpower.reviews
+
+
+    if !@vote.save
+      flash[:alert] = "You can only vote once!"
+    end
+  end
+
+  def destroy
```

Comment:
this has some security problems - we should check for the last vote that the user made for the specific review

Otherwise we could inadvertently delete someone else's vote!